### PR TITLE
Made modifications.

### PR DESCRIPTION
**Pull request by mhmatthewhugley**
Being able to upgrade a bow into crossbow was added. Uses crafting to upgrade they just add a iron ingot and tripwire hook. This is the exact material difference besides 1 other material, if you were going to make a crossbow first instead of a bow.

Chainmail armor to iron was added. Uses iron ingot in smithing table.

Gold to chainmail armor was modified instead of using cobblestone it uses a minecraft:chain which is craftable and can be found.

Gold to iron armor was removed. (Because it would skip the chainmail stage) (Based on the "Protection value" section and the "Tiers" section of this page "https://minecraft.fandom.com/wiki/Armor")

Wooden to stone tools was modified, now it uses the miencraft:stone instead of cobblestone. (Because that is more fitting with the look of stone tools.)

### CODE REVIEW

**Comment by mhmatthewhugley** [comment]
Also the README.md was modified to include the new features and features that was missing. And adding my name and role and everything to the contributor section. And other changes might have been made. I think I listed everything though.

File: upgrade_everything_v1.5.2/data/upgrade-everything/recipes/wooden-to-stone-hoe.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-		"item": "minecraft:stone_hoe"
-	}
-}
+{
+	"type": "minecraft:smithing",
+	"base": {
+		"item": "minecraft:wooden_hoe"
+	},
+	"addition": {
+		"item": "minecraft:stone"
```
Comment (NoComment1105) [inline upgrade_everything_v1.5.2/data/upgrade-everything/recipes/wooden-to-stone-hoe.json]: I'd prefer cobblestone to stay used here, due to cobblestone being what is used to craft stone toosl in the first place
```suggestion
                "item": "minecraft:cobblestone"
```

**Review by NoComment1105 — CHANGES_REQUESTED** [review]
Just a single request. But otherwise looking good!

**Comment by mhmatthewhugley** [comment]
Yes that makes a lot of sense I guess I just forgot about that when I made modifications.

**Comment by NoComment1105** [comment]
You need to do the other 4 too lmao

**Comment by mhmatthewhugley** [comment]
Did I do it all right?

**Review by NoComment1105 — APPROVED** [review]
All good, will merge tonight

**Comment by mhmatthewhugley** [comment]
I also updated the readme to change it from stone to cobblestone.

**Review by NoComment1105 — APPROVED** [review]

**Comment by mhmatthewhugley** [comment]
My server would not start up unless it was put in a crafting directory so that is what those changes are for. (Java 17) (Paper MC 402)

**Comment by NoComment1105** [comment]
The reason it wasn't starting is due to the recipe file being wrong. The reason it loads now is because that director does nothing

**Comment by mhmatthewhugley** [comment]
I think I fixed the recipe file.

**Comment by mhmatthewhugley** [comment]
I was able to fix it but by using crafting instead of the smithing table it would not keep enchantments.

**Comment by NoComment1105** [comment]
I'm gonna have to test this before merge because I've not no idea what's actually going on

**Comment by mhmatthewhugley** [comment]
Sounds good. I expected that. I am putting the bow to crossbow back on the smithing table cause if I did it with the crafting table it would not keep enchantments

**Comment by mhmatthewhugley** [comment]
How did your christmas go? How did testing it go? Does anything need to be changed?

**Comment by NoComment1105** [comment]
Haven't had time

**Comment by mhmatthewhugley** [comment]
When you test it let me know how it goes and if anything needs to be changed.

**Comment by mhmatthewhugley** [comment]
> Haven't had time

Whenever you can that would be great so I can archive my repository.

**Pull request closed without merging**